src/directives/uiSref.ts

```diff
@@ -129,6 +130,7 @@ export class UISref implements OnChanges {
     this._parent = parent;
 
     this._statesSub = _router.globals.states$.subscribe(() => this.update());
+    this._statesSub = _router.globals.params$.subscribe(() => this.update());
```

Comment:
I'm guessing this was supposed to be _paramsSub

Reply:
That was definitely my intention 😅  Fixed it! (https://github.com/ui-router/angular/pull/976/commits/0ebf4c1cebd081225911f349cae8bc748d008c06)